### PR TITLE
PT-401 Display rescueCarbs with specific layout in Daily view

### PR DESCRIPTION
**Pull request by franck-fourel**
- display rescuecarbs as light red meal
- Fix translations
- add doc about translations

### CODE REVIEW

**Comment by franck-fourel** [comment]
This PR does not used the final releases of viz and tideline; it's still pointing to the github branches.

File: locales/en/translation.json [hunk 1]
```diff
@@ -460,6 +460,7 @@
   "Remember me": "",
   "Remove": "",
   "Remove People": "",
+  "rescuecarbs": "Rescuecarbs",
```
Comment (yvon-dblg) [inline locales/en/translation.json]: Warn: Tideline use "rescuecarbs" fot the translation, viz: "Rescuecarbs".

Comment (franck-fourel) [reply locales/en/translation.json]: let's stick to 'rescuecarbs'

**Comment by franck-fourel** [comment]
Still neet to update package.json once viz and tideline tags are generated

**Comment by franck-fourel** [comment]
Final version with updated packages for viz and Tideline

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 Blip is the web front end for YourLoops system.
 It is based on Tidepool Blip 1.20.
 
+## unreleased
```
Comment (yvon-dblg) [inline CHANGELOG.md]:
```suggestion
## [0.7.0] - 2019-10-11
```

Comment (franck-fourel) [reply CHANGELOG.md]: no update for now

**Pull request merged**